custom_components/easee/const.py

```diff
@@ -119,14 +132,16 @@
         ],
         "units": None,
         "convert_units_func": None,
+        "device_class": DEVICE_CLASS_SIGNAL_STRENGTH,
         "icon": "mdi:wifi",
```

Comment:
I suggest DEVICE_CLASS_CONNECTIVITY - import from homeassistant.components.binary_sensor. You can test with Icon: None also

Reply:
Yes, better. Changed now. Thanks.